### PR TITLE
front: handle accents in STDCM vias autocomplete

### PR DESCRIPTION
**Pull request by leovalais**

### CODE REVIEW

File: front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx [hunk 1]
```diff
@@ -22,6 +22,18 @@ function formatChCode(chCode: string) {
   return chCode === '' ? 'BV' : chCode;
 }
 
+// Normalizes a string by removing accents and converting to lowercase
+//
+// Accent removal is done using `String.normalize` which converts accented characters to their
+// decomposed form (e.g. é -> e + ´). We then remove all characters in the range of U+0300 to U+036f
+// which contains all unicode combining diacritical marks (https://en.wikipedia.org/wiki/Combining_Diacritical_Marks).
+function normalized(s: string): string {
```
Comment (SarahBellaha) [inline front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx]: Maybe we should have this function in `front/src/utils/strings.ts` ?

**Review by SarahBellaha — APPROVED** [review]
LGTM, Thanks ! 💯

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested, thank you !

File: front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx (hunk 1 above)
Comment (SharglutDev) [reply front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx]: Agreed. You could also use the syntax `/** */` for the comment so we can see it when hovering the function.

Comment (leovalais) [reply front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx]: Done.

**Pull request merged**